{{ cookiecutter.repo_name }}/tox.ini

```diff
@@ -1,12 +1,19 @@
 [tox]
 isolated_build = true
-envlist = py{38,39,310,311,312}-django{32,42}-psycopg2,py312-django42-psycopg3,report
+envlist =
+    py{38,39,310,311,312}-django32-psycopg2
+    py{38,39,310,311,312}-django42-psycopg2
```

Comment:
I retained the “no intermediate versions” setup here. It seems they were removed in 695f0ad9299255d643870905b34894790e6e6b58, I guess to speed up CI? Personally, I prefer the confidence of running on all supported versions.

Reply:
Yea it just became very lengthy. I wanted to split out many of the combinations across parallel circle runner.

Happy to add this back in a follow up though